director/api/public.py

```diff
@@ -1,9 +1,40 @@
-from flask import Blueprint
-from director.model import User
-
+from flask import Blueprint, Response,  request, make_response
+from director.model import User, Terminal
+from director.authentication import  token_required , generate_Jwt, Authenticator
+secret = 'pass'
+valid_password = 'pass'
```

Comment:
Saving variables like this generally frowned upon. Maybe move them inside the methods that use them?
If they are global stuff that's needed in the project, you can also save them in the `app.config`!